internal/document/block.go

```diff
@@ -135,6 +135,19 @@ func (b *CodeBlock) Value() []byte {
 	return b.value
 }
 
+func (b *CodeBlock) Category() string {
+	return b.Attributes()["category"]
+}
+
+func (b *CodeBlock) IsExcludedFromRunAll() bool {
```

Comment:
```suggestion
func (b *CodeBlock) ExcludeFromRunAll() bool {
```

Consistent naming convention 😉